demos/python_demos/bert_question_answering_demo/bert_question_answering_demo.py

```diff
@@ -163,16 +93,16 @@ def main():
 
     # load vocabulary file for model
     log.info("Loading vocab file:\t{}".format(args.vocab))
-    with open(args.vocab, "r", encoding="utf-8") as r:
-        vocab = {t.rstrip("\n"): i for i, t in enumerate(r.readlines())}
+    vocab = load_vocab_file(args.vocab)
     log.info("{} tokens loaded".format(len(vocab)))
 
     # get context as a string (as we might need it's length for the sequence reshape)
-    context = get_context(args.input)
+    paragraphs = get_paragraphs(args.input.split(','))
```

Comment:
IMO, it would be better to mark the `--input` parameter with `action='append'`, thus turning it into a list. Commas can technically appear within URLs, and it's easier to read multiple URLs when they're in separate arguments.

Reply:
Then it is required to make the same for all 'list' args (input_names for example).

Reply:
You are right. I found wiki page url with commas ;(
I will use append for --input despite strange final cmd line 
demo.py --input url1 --input url2
looks like second input overrides the first one.